### PR TITLE
[SNAP-31] Close browser tab after generating a snapshot.

### PR DESCRIPTION
**Pull request by cafuego**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -387,6 +387,9 @@ app.post('/snap', [
           } else {
             await page.pdf(pdfOptions);
           }
+          
+          // Close the browser tab.
+          await page.close();
```
Comment (cafuego) [inline app/app.js]:
```suggestion
          await browserContext.close();
```

Comment (cafuego) [reply app/app.js]: Ha! Yes, `page.close();` then :-)

Comment (rupl) [reply app/app.js]: Still, if there are resources piling up I should look into any possibility of GC or otherwise releasing dormant memory.

Comment (rupl) [reply app/app.js]: The docs show that using the BrowserContext as an instance is possible: https://pptr.dev/#?product=Puppeteer&version=v1.11.0&show=api-class-browsercontext

It says 1.5.0+ and we're 1.8.0 right now. I'll give this a shot.

Comment (rupl) [reply app/app.js]: well shucks. I shuffled stuff around to use an incognito context but it still seems like processes are piling up. I'm using `docker top [img]` to monitor this so I think i'm doing it not wrong. 

I found a couple issues relating to zombie processes and apparently you can get the pid when you disconnect and then kill it manually in node.

**Comment by cafuego** [comment]
Closing in favour of https://github.com/UN-OCHA/tools-snap-service/pull/26

**Pull request closed without merging**